### DISCUSSION TITLE
Why is Github calling my repo 70%+ hack?

### DISCUSSION THREAD

**Discussion by yakun-hu** [community/community#42161] [Repositories]
### Select Topic Area

Product Feedback

### Body

This is pretty basic code; I have AJAX/JS and PHP. However, Github is unable to read the JS, so it's saying I have 70% hack, 30% PHP, when it should be 70% JS, 30% PHP: https://github.com/yakun-hu/flare

**Comment by getlost01** [comment]
Hack files can also use the `.php`extension. Since Linguist considers each file independently and that file only contains HTML code, it's going to be difficult (impossible?) for Linguist to understand that it is not a Hack file. You can use an [override](https://github.com/github/linguist#overrides) to change that behavior though.
